tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>if <code>detlabels</code> is None, you could use the keys as labels, so we avoid to build the trivial dict <code>x:x</code>.</p><p>please use keyword arguments for all inputs, so people don't have to look at the help of <code>plot_focalplane</code></p><p> </p><p>For the color, what about using <code>endswith("A")</code>  instead of enumerating?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Ok, this is cleaned up.</p>